### PR TITLE
[FE] [MODIFY] RedirectUser branch on master (ReviewBE, RedirectUserFE, CSSLoginBtn)

### PR DESCRIPTION
**Pull request by Alban-de-tourtier**
### 🛠 Changes being made

.Added Review back end endpoint
.Change router restoweb & visitorWeb to protect page when user is logged or not
.Modify login button style Header component

### ✨ What's the context?

PLD ticket and correction(style missed)

### 🧪 Test plan

I test everything and Josi make a review so we can merge it without problem

### 🏎 PR checklist

1. Read the ticket
- [X] Read the related ticket on Jira and understand the requirements for this ticket. In case anything is unclear, ask the ticket creator or ticket executor.
2. Matching PR description
- [X] Check that the PR description on GitHub is aligned with the ticket description on Jira.
3. Review the changes on GitHub
- [X] is the current version of the destination branch (mostly main) merged into the PR branch?
- [X] does it fulfill the definition of done for the ticket?
- [X] were the tests adjusted or extended for the changed behavior?
- [X] was the documentation adjusted or extended for changed behavior?
- [X] does it align with the team code guidelines?
- [X] check readability of code - is it easy to understand what is happening in the code snippets?
- [X] proper naming
- [X] is it logical?
- [X] are there any performance issues?
- [X] could it be improved / simplified?
4. Execute locally / in testing environment
- [X] Execute the code locally and check if there are any issues. Test whether the new behavior from the ticket acts as expected. Try to think of edge cases and test them.
5. Run tests
- [X] Run the tests locally and check that all of them succeed.
6. Linting & Formatting
- [X] Check that the changes didn’t cause any linting or formatting warnings of errors.
7. Submit Feedback
- [X] Approve if all of the following points fulfilled:
   - all previous steps did not reveal any issues
   - there are no open comments on the PR

- [X] otherwise, request changes with specific comments (constructive) on what to change!

### CODE REVIEW

**Review by JosefineM2000 — CHANGES_REQUESTED** [review]
Please also update the PR title according to our commit policy and write a propper PR description instead of leaving the template as is.

File: packages/restoWeb/src/router/Router.tsx [hunk 1]
```diff
@@ -19,9 +20,32 @@ import ResetPassword from "@src/pages/ResetPasswordPage";
 import MyAccountPage from "@src/pages/MyAccountPage";
 
 const MVPRouter = () => {
+  const [isUserTokenSet, setIsUserTokenSet] = useState<boolean>();
+  const userToken = localStorage.getItem('user');
+
+  const checkUserToken = () => {
+    if (userToken === null) {
```
[the remark this answers is not in the document]
Comment (Alban-de-tourtier) [reply packages/restoWeb/src/router/Router.tsx:27]: Im doing that in the use effect

File: packages/visitorWeb/src/router/Router.tsx [hunk 2]
```diff
@@ -13,8 +13,31 @@ import IntroPage from "@src/pages/IntroPage";
 import ResetPassword from "@src/pages/ResetPasswordPage/ResetPassword";
 
 const MVPRouter = () => {
+  const [isUserTokenSet, setIsUserTokenSet] = useState<boolean>();
+  const userToken = localStorage.getItem('user');
+
+  const checkUserToken = () => {
+    if (userToken === null) {
```
[the remark this answers is not in the document]
Comment (Alban-de-tourtier) [reply packages/visitorWeb/src/router/Router.tsx:20]: Im doing that in the use effect

**Review by JosefineM2000 — APPROVED** [review]
